ccd-definition/AuthorisationCaseEvent/CareSupervision/AuthorisationCaseEvent-nonprod.json

```diff
@@ -45,5 +45,26 @@
         "CRUD": "R"
       }
     ]
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "addGatekeepingOrder",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-publiclaw-judiciary"
```

Comment:
should be fine as ticket specified this is only for judge role but we should check if any users still have gatekeeper role (it should be deprecated/removed)

Reply:
leave as is but check before release if we need gatekeeper permission